src/disjoint_set.jl

```diff
@@ -109,15 +143,15 @@ end
     intmap::Dict{T,Int}
     internal::IntDisjointSets
```

Comment:
This is now an abstract type. I suspect you want to add another parameter to `DisjointSets`.